docs/docsite/rst/getting_started/get_started_playbook.rst

```diff
@@ -0,0 +1,69 @@
+.. _get_started_playbook:
+
+*******************
+Creating a playbook
+*******************
+
+Playbooks are automation blueprints, in ``YAML`` format, that Ansible uses to deploy and configure managed nodes.
+Playbooks define the order in which Ansible performs actions, from top to bottom, and contain the following objects:
+
+Play
+   An ordered list of tasks that maps to managed nodes in an inventory.
+
+Task
+   A list of one or more modules that defines the actions that Ansible performs.
+
+Module
+   A unit of code or binary that Ansible runs on managed nodes.
+   Ansible modules are grouped in collections with a fully qualified collection name (FQCN) for each module.
```

Comment:
The introductory sentence `"Playbooks define..., and contain the following objects"` might make it sound like the unit of code is part of the playbook. Maybe the first sentence could be something like
`The name to a unit of code or binary that Ansible runs on managed nodes`?

Reply:
I've reworked the introductory sentence for a previous comment so I think that resolves this issue about implying that the unit of code itself is part of the playbook. Thank you for the feedback too!